vault/resource_pki_secret_backend_cert.go

```diff
@@ -245,24 +250,23 @@ func pkiSecretBackendCertCreate(d *schema.ResourceData, meta interface{}) error
 	d.Set("serial_number", resp.Data["serial_number"])
 	d.Set("expiration", resp.Data["expiration"])
 
+	err = pkiSecretBackendCertSynchronizeRenewPending(d)
```

Comment:
rather than shadowing `err` here, would you doing the assignment within the `if` statement?

```go
if err := ... ; err != nil {...}
```